Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "FirebaseManager",
+    platforms: [
+        .iOS(.v14),
+        .macOS(.v10_15)
```

Comment:
We support iOS 14 but only the most recent version of macOS? Do you know whether there is a difference, so the package wouldn't work on a previous version? Or should we should just go for iOS 15, so we could use more recently supported functions as well?

Reply:
Yes, changed macOS also to 14. Should be fine since the referenced packages support way older